src/core/thread/network_data_leader_ftd.cpp

```diff
@@ -60,9 +60,15 @@ RegisterLogModule("NetworkData");
 
 Leader::Leader(Instance &aInstance)
     : LeaderBase(aInstance)
+#if OPENTHREAD_CONFIG_BORDER_ROUTER_SIGNAL_NETWORK_DATA_FULL
+    , mIsClone(false)
+#endif
     , mWaitingForNetDataSync(false)
     , mContextIds(aInstance)
     , mTimer(aInstance)
+#if OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+    , mIgnoreRegisteration(false)
```

Comment:
```suggestion
    , mIgnoreRegistration(false)
```

Reply:
Thanks for noticing this. Fixed all cases on this in new push.